src/components/Integrations/BigQuery/ControlPanel/InputSpanForm.js

```diff
@@ -0,0 +1,143 @@
+import React, { useState, useEffect, useCallback } from 'react';
+import { Formik, Field, Form, FieldArray } from 'formik';
+import { useIntl } from 'react-intl';
+import { validateEmail } from '../../../../validations';
+import { isUniqueArray } from '../../../../utils';
+import { Loading } from '../../../Loading/Loading';
+
+export const InputSpanForm = () => {
+  const [initialValues, setInitialValues] = useState([]);
+  const [isLoading, setIsLoading] = useState(false);
+  const [errorMessage, setErrorMessage] = useState('');
+
+  const intl = useIntl();
+  const _ = useCallback(
+    (id, values) => {
+      return intl.formatMessage({ id: id }, values);
+    },
+    [intl],
+  );
+
+  useEffect(() => {
+    const fetchInfo = () => ['email1@gmail.com', 'email2@gmail.com'];
+    setIsLoading(true);
+    setTimeout(() => {
+      const data = fetchInfo();
+      setInitialValues([...data, '']);
+      setIsLoading(false);
+    }, 2000);
+  }, [setInitialValues]);
```

Comment:
**`setInitialValues`** it is not necessary here. 

You can use `**useTimeout()**` hook. 
**`const createTimeout = useTimeout();`**

Reply:
suggest accepted and applied!

Reply:
Hecho, resuelvo la conversación...